src/pytest_plugins/consume/hive_simulators/ruleset.py

```diff
@@ -5,6 +5,49 @@
 Remove this file afterwards.
 """
 
+from typing import Dict
+
+from ethereum_test_forks import Cancun, Fork, Osaka, Prague
+
+
+def get_blob_schedule_entries(fork: Fork) -> Dict[str, int]:
+    """
+    Generate blob schedule entries for each fork (and respective parent forks).
+
+    Adds the following entries to the ruleset for the given fork (and parent forks):
+        HIVE_{FORK}_BLOB_TARGET: target_blobs_per_block()
+        HIVE_{FORK}_BLOB_MAX: max_blobs_per_block()
+        HIVE_{FORK}_BLOB_BASE_FEE_UPDATE_FRACTION: blob_base_fee_update_fraction()
+    """
+    entries = {}
+
+    def add_fork_entries(fork):
```

Comment:
Am I missing something or is `add_fork_entries` not called on the top level function?